src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -5,8 +5,7 @@
 import com.pathplanner.lib.auto.PIDConstants;
```

Comment:
With the robot map containing so many details now - maybe we should split it into a folder with multiple files? 
One of the robots, one for products details one for field, or stuff like that?

Reply:
for every subsystem to make <subsystem>constants?

Reply:
Generally, I agree, I think it also should be read from JSON, but it's outside the scope the feature / CR